config/settings.yml

```diff
@@ -3,9 +3,19 @@
     :blacklisted_event_names: []
     :event_handling:
       :event_groups:
+:ems_refresh:
+  :scvmm:
+    :refresh_interval: 15.minutes
 :http_proxy:
   :scvmm:
     :host:
     :password:
     :port:
     :user:
+:log:
+  :level_scvmm: info
+:workers:
+  :worker_base:
+    :queue_worker_base:
+      :ems_refresh_worker:
+        :ems_refresh_worker_microsoft: {}
```

Comment:
I feel like this one is wrong, but this is the value from core...what is the correct value?

Reply:
Isn't that what most of them are by default?

Reply:
most of the attributes in this provider say "scvmm" and this one says "microsoft"

Reply:
Internally the class names use "Microsoft", e.g. `ManageIQ::Providers::Microsoft::InfraManager::RefreshWorker`. I don't know the config setting very well, but maybe it needs to match?

Reply:
The reason for seeing both here is that the class name is `ManageIQ::Providers::Microsoft` like @djberg96 mentions and the `InfraManager`'s `ems_type` is `scvmm`

The settings that key off of the ems_type are `scvmm` and the ones which use the class path are `microsoft`.

I think this provider highlights our inconsistency between `vendor::manager_type` and `product::*managers`

For example, we should have either:
```
ManageIQ::Providers::Microsoft::CloudManager => azure
ManageIQ::Providers::Microsoft::InfraManager => scvmm
```

or

```
ManageIQ::Providers::Scvmm::InfraManager
ManageIQ::Providers::Azure::CloudManager
```

But we have a mix of both :smile: 

It will become more important for us to nail this down when we start adding in other "services" like kubernetes-type services to existing providers like amazon.

E.g.:
```
ManageIQ::Providers::Amazon::CloudManager => EC2
ManageIQ::Providers::Amazon::StorageManager::Ebs => EBS
ManageIQ::Providers::Amazon::ContainerManager => EKS
```

This looks like it fits the `vendor::service` model more.

Maybe we should be using the "vendor" for more of these settings here?

Reply:
I do like
```
ManageIQ::Providers::Microsoft::CloudManager => azure
ManageIQ::Providers::Microsoft::InfraManager => scvmm
```

Reply:
what should we do for open-source projects like Kubernetes that do not have a single vendor?

Reply:
That's a good point maybe vendor is the wrong term, it is essentially just the provider name.  In a lot of cases the provider name is the vendor but you're right in the case of a project like k8s it the project name.

Reply:
Cool...so then this is `ems_refresh_worker_microsoft` is currently correct and we can change later.